Sprint-1/errors/1.js

```diff
@@ -1,4 +1,6 @@
 // trying to create an age variable and then reassign the value by 1
 
-const age = 33;
+/* The value of a constant cannot change through re-assignment,
+we can use var or let insted*/
+var age = 33;
```

Comment:
Try to avoid using `var`; it could have weird side effect if not used properly. Use `let` instead to declare local variables.